NOTES.ANDROID

```diff
@@ -0,0 +1,66 @@
+
+ NOTES FOR THE ANDROID PLATFORMS
+ ===============================
+
+ Requirement details
+ -------------------
+
+ Beside basic tools like perl and make you'll need to download Android
+ NDK. It's available for Linux, MacOS X and Windows, but only Linux
```

Comment:
"MacOS X" may be better as "Mac OS X" or "macOS"? See https://en.wikipedia.org/wiki/MacOS

Reply:
Or maybe "OS formerly known as something with Mac and X in its name"... But jokes aside, sure, I'll fix...